src/Entity/RecoveryCode.php

```diff
@@ -61,4 +66,14 @@ public function setSecret(string $secret): void
     {
         $this->secret = $secret;
     }
+
+    public function getDownloaded(): bool
```

Comment:
```suggestion
    public function isDownloaded(): bool
```
For bools I'd prefer to use issers or hassers